cmd/utils/flags.go

```diff
@@ -138,6 +138,11 @@ var (
 		Name:  "olympic",
 		Usage: "Use olympic style protocol",
 	}
+	EthVersionFlag = cli.IntFlag{
+		Name:  "eth",
+		Value: 61,
```

Comment:
I didn't enable it because the plan was originally to have it in master too, with it being enabled in develop only. Since it will miss master, this needs to be bumped to 62 for develop :)